doc/cephadm/troubleshooting.rst

```diff
@@ -123,23 +123,54 @@ If you have enabled logging to files, you can see a cephadm log file called
 Gathering log files
 -------------------
 
-Use journalctl to gather the log files of all daemons:
+General Procedure for Gathering Log Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. note:: By default cephadm now stores logs in journald. This means
-   that you will no longer find daemon logs in ``/var/log/ceph/``.
+Use ``journalctl`` to gather the log files of all daemons.
 
-To read the log file of one specific daemon, run::
+.. note:: Since Octopus, cephadm stores logs in journald by default. 
+          Since Octopus, daemon logs are not stored in in 
+          ``/var/log/ceph/``.
+
+A command of the following form (``journalctl`` run by itself without any
+flags) returns a list of all journal events:
+
+.. prompt:: bash #
+
+  journalctl
```

Comment:
please don't let users run this. This is super irritating. I'd suggest to only run journalctl for specific units.

Reply:
I've straight-up removed this.